### DISCUSSION TITLE
win or go home

### DISCUSSION THREAD

**Discussion by tower20021202** [1101-GS4719-C/Python-Community-GS4719-C#1414] [General]
今天看p聯盟台新夢想家對高雄鋼鐵人的比賽，兩對誰贏就能晉級季後賽，結果第二節書豪哥就被對面洋將手肘撞到送醫檢查，此外在中場時主播更説賣卡洛今日罰球六罰六中外帶一ko，雖說對手不是故意的，但最後在林書豪不在的情況下，鋼鐵人最終不敵夢想家被淘汰了，在林書豪籃球生涯的晚期，還是希望能多看到他比賽

**Comment by brian431** [comment]
這場前面林書豪就一直被很大動作的肢體動作影響到了，不過最後那個肘擊應該不是故意的沒錯啦。只能說有夠衰。

**Comment by PaulSu0905** [comment]
林書豪確實是一位優秀的籃球選手，他的表現和領導能力經常能帶給球迷們精彩的比賽。希望他能在籃球生涯的晚期繼續保持健康，並給球迷們帶來更多精彩的比賽。也希望他有機會與弟弟同隊。
